.github/workflows/release.yml

```
@@ -214,17 +214,17 @@ jobs:
                   name: contract-artifacts
 
             - name: Verify contracts
-              run: eval yarn hardhat --network ${{ needs.prepare.outputs.release_network }} ${{ needs.prepare.outputs.release_verify_cmd }} ${{ needs.prepare.outputs.release_version }}
+              shell: bash
+              run: |
+                yarn hardhat --network ${{ needs.prepare.outputs.release_network }} ${{ needs.prepare.outputs.release_verify_cmd }} ${{ needs.prepare.outputs.release_version }}
```

https://wiki.polygon.technology/docs/develop/hardhat/#deploying-on-polygon-network
```suggestion
                yarn hardhat --network polygon_${{ needs.prepare.outputs.release_network }} ${{ needs.prepare.outputs.release_verify_cmd }} ${{ needs.prepare.outputs.release_version }}
```